image.go

```diff
@@ -53,17 +68,31 @@ func (i *containerImageRef) NewImage(sc *types.SystemContext) (types.Image, erro
 }
 
 func (i *containerImageRef) NewImageSource(sc *types.SystemContext, manifestTypes []string) (src types.ImageSource, err error) {
```

Comment:
When maintaining two years hence, searching for 'i' will be a bear through out this function.  Any chance in your next pass through you could change this to at least 'img' to make it more markable?   I'd really love 'containerImage', but willing to settle for at least a few more characters.